changelog/wave_1.12.3.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: Wave v1.12.3
```

Comment:
@justinegeffen, is this part of the PR?

Reply:
I don't actually know how this got into the PR - must have been merged. I'll delete it.